app/models/customer.rb

```diff
@@ -13,6 +13,11 @@ class Customer < ActiveRecord::Base
     where('name LIKE :terms OR email LIKE :terms OR identification LIKE :terms', terms: '%' + terms + '%')
   }
 
+  scope :only_active, ->(boolean = true) {
```

Comment:
had to do this, since simply using a checkbox for `active` field , when checked is active, when unchecked is inactive, because ruby always add a hidden field with the opposite value.